docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md

```diff
@@ -28,18 +28,16 @@ The enhanced LDAP integration adds additional functionality on top of the [LDAP
 
 > To control user access with role-based permissions, refer to [role-based access control]({{< relref "../../../../administration/roles-and-permissions/access-control" >}}).
 
-## LDAP group synchronization for teams
+## LDAP group synchronization
 
-With enhanced LDAP integration, you can set up synchronization between LDAP groups and teams. This enables LDAP users that are members
-of certain LDAP groups to automatically be added or removed as members to certain teams in Grafana.
+With enhanced LDAP integration, you can set up synchronization between LDAP groups and Grafana teams and role-based access control roles. This enables LDAP users that are members
+of certain LDAP groups to be automatically added to teams and granted roles in Grafana.
```

Comment:
nit: i do not know if i am reading this incorrectly. The wording of `to team and granted roles` seems like it is only specific roles. Is this correct?

```suggestion
of certain LDAP groups to be automatically added to teams and granted roles in Grafana.
```

Reply:
Thanks for the feedback, I'll try to reword this.